packages/firestore/src/platform/browser/webchannel_connection.ts

```diff
@@ -27,9 +27,9 @@ import {
   EventTarget,
   StatEvent,
   Event,
-  Stat,
-  FetchXmlHttpFactory
-} from '@firebase/webchannel-wrapper';
+  FetchXmlHttpFactory,
+  Stat
+} from '@firebase/webchannel-wrapper/webchannel-blob';
```

Comment:
Nit: requiring a change in the import is a breaking change, which does not align with the point release. We do say that webchannel-wrapper is for internal use only, which is why I only labeled this a nit.